adapters/triplelift_native/triplelift_native.go

```diff
@@ -50,12 +50,6 @@ func processImp(imp *openrtb2.Imp, request *openrtb2.BidRequest) error {
 	// get the triplelift extension
 	var ext ExtImp
 	var tlext openrtb_ext.ExtImpTriplelift
-	var siteCopy openrtb2.Site
-	var extData ExtImpData
```

Comment:
Copying `request.Site` into `siteCopy` and `ext.Data` into `extData` created overhead and, given that the `processImp` function is only interested in reading and not overwriting thesevalues, no shared memory issues will come if we just use the originals. Let me know if you agree with their removal.